contents/data-display/banners/cookie/index.tsx

```diff
@@ -16,24 +16,14 @@ const Banner: FC = () => {
     <Card variant="outline" maxW="md" shadow="sm">
       <CardBody>
         <Grid
-          templateAreas={`
+          templateAreas={{
+            base: `
```

Comment:
I thought you don't need`base` because it doesn't resize to fit the screen width.

```suggestion
          templateAreas={`
```

Reply:
As you said.
It is my mistake.
Thanks for reviewing it.

Reply:
As you said.
It is my mistake.
Thanks for reviewing it.